### ISSUE TITLE
Game crashes upon opening continue screen

### ISSUE THREAD

**Issue by dza205** [egocarib/CavesOfQud-QudUX-v2#15]
```
Mono path[0] = 'C:/Program Files (x86)/Steam/steamapps/common/Caves of Qud/CoQ_Data/Managed'
Mono config path = 'C:/Program Files (x86)/Steam/steamapps/common/Caves of Qud/MonoBleedingEdge/etc'
Initialize engine version: 2020.3.19f1 (68f137dc9bbe)
[Subsystems] Discovering subsystems at path C:/Program Files (x86)/Steam/steamapps/common/Caves of Qud/CoQ_Data/UnitySubsystems
GfxDevice: creating device client; threaded=1
Direct3D:
    Version:  Direct3D 11.0 [level 11.1]
    Renderer: NVIDIA GeForce GTX 1660 SUPER (ID=0x21c4)
    Vendor:   (null)
    VRAM:     3072 MB
    Driver:   27.21.14.5671
Begin MonoManager ReloadAssembly
- Completed reload, in  0.218 seconds
<RI> Initializing input.

<RI> Input initialized.

<RI> Initialized touch support.

UnloadTime: 0.518800 ms
<color=#AAAAAA>[Info] GameLogger initialized...</color>
<color=orange>Logger..cctor</color>
<color=red></color>
<color=#AAAAAA>[Info] TraceLogger initialized...</color>
<color=orange>Logger..cctor</color>
<color=red></color>
<color=#AAAAAA>[Info] Starting up logger...</color>
<color=orange>GameManager.Awake</color>
<color=red></color>
<color=#AAAAAA>[Info] Getting player 0...</color>
<color=orange>GameManager.Awake</color>
<color=red></color>
<color=#AAAAAA>[Info] Got player 0...</color>
<color=orange>GameManager.Awake</color>
<color=red></color>
Startup time: jueves, 12 de mayo de 2022 at 22:00:51
Version: 2.0.203.36
Platform: WindowsPlayer
System Language: Spanish
Unity Version: 2020.3.19f1
Unity Reported Version: 2.0.4
Requesting stats...
INFO - Subscribed workshop items: 8
INFO - Loading Colors.xml
INFO - Done Loading Colors.xml in 14ms
MODWARN [2400127933] - Mod using config.json, please convert to manifest.json and check out https://wiki.cavesofqud.com/Modding:Overview for other options to set
INFO - Enabled mods: Autoget Anything, Dynamic Background Color, 1Percent Loot Drops Expanded, QudUX, Suppress Scavenging Popups
INFO - Subscribed workshop items: 8
MODWARN [2400127933] - Mod using config.json, please convert to manifest.json and check out https://wiki.cavesofqud.com/Modding:Overview for other options to set
INFO - Enabled mods: Autoget Anything, Dynamic Background Color, 1Percent Loot Drops Expanded, QudUX, Suppress Scavenging Popups
INFO - Building script mods
Fallback handler could not load library C:/Program Files (x86)/Steam/steamapps/common/Caves of Qud/CoQ_Data/Mono/data-16B55AD0.dll
Fallback handler could not load library C:/Program Files (x86)/Steam/steamapps/common/Caves of Qud/CoQ_Data/Mono/data-2764BBB8.dll
QudUX: Applying pre-game patches...
    QudHistoryFactory...    Patched successfully.
    Scores...               Patched successfully.
    XRLCore.NewGame...      Patched successfully.
    AbilityManager...       Patched successfully.
    Look...                 Failed. This patch may not be compatible with the current game version. The option to mark legendary creature locations in journal won't be available from the Look UI.
    GameObject.Move...      Patched successfully.
    Campfire...             Failed. This patch may not be compatible with the current game version. The game's default cooking UI pop-ups will be used instead of QudUX's revamped screens.
    MagneticPulse...        Patched successfully.
    Physics.HandleEvent...  Failed. This patch may not be compatible with the current game version. Some particle text effects may not be shown when movement is prevented.
INFO - Done Building script mods in 2970ms
INFO - Resetting static caches
INFO - Loading genders
INFO - Done Loading genders in 3ms
INFO - Loading PronounSets.xml
INFO - Done Loading PronounSets.xml in 4ms
INFO - Done Resetting static caches in 55ms
INFO - Loading Colors.xml
INFO - Done Loading Colors.xml in 1ms
WARN - Multiple debug commands defined for: hide
WARN - Multiple debug commands defined for: clear
WARN - Multiple debug commands defined for: write
WARN - Multiple debug commands defined for: writeline
WARN - Multiple debug commands defined for: help
WARN - Multiple debug commands defined for: select
WARN - Multiple debug commands defined for: list
WARN - Multiple debug commands defined for: call
Sidebar swap to left
Starting game thread...
Data path: C:/Program Files (x86)/Steam/steamapps/common/Caves of Qud/CoQ_Data/StreamingAssets\Base
Save path: C:/Users/danid/AppData/LocalLow/Freehold Games/CavesOfQud
Starting core...
Started!
Cleaning up...
Starting up XRL...
Initialize Genders and Pronoun Sets...
Initialize Name Styles...
INFO - Loading Naming.xml
exTextureInfo must be instantiated using the ScriptableObject.CreateInstance method instead of new exTextureInfo.
Initilization complete...
PlayFab player id: c65565d9-d7bd-4c2c-af47-1eea74fde706
The header Accept-Encoding is managed automatically, setting it may have no effect or result in unexpected behavior.
Sidebar swap to right
INFO - Done Loading Naming.xml in 9ms
Loading World Blueprints...
Loading Help...
Init Pathfinder...
Init Console UIs...
Looker INIT!
INFO - Loading ZoneTemplates.xml
INFO - Done Loading ZoneTemplates.xml in 8ms
INFO - Loading Quests.xml
INFO - Done Loading Quests.xml in 7ms
INFO - Loading ObjectBlueprints.xml
INFO - Loading Mutations.xml
INFO - Done Loading Mutations.xml in 4ms
PlayFab login success.
PlayFab token expires 05/13/2022 20:00:55
INFO - Done Loading ObjectBlueprints.xml in 982ms
INFO - Dispatch LoadBlueprint
INFO - Initialize tinkering
INFO - Done Initialize tinkering in 2ms
INFO - Loading Mods.xml
INFO - Done Loading Mods.xml in 3ms
INFO - Loading medication names
INFO - Done Loading medication names in 21ms
INFO - Loading PopulationTables.xml
MODWARN [1Percent Loot Drops Expanded] - XmlDataHelper:: file:///C:/Program Files (x86)/Steam/steamapps/workshop/content/333640/2400127933/PopulationTables.xml line 5 char 54
Unused attribute "Amount" detected.
INFO - Done Loading PopulationTables.xml in 23ms
INFO - Initialize liquids
INFO - Done Initialize liquids in 68ms
INFO - Loading Factions.xml
INFO - Done Loading Factions.xml in 5ms
INFO - Done Dispatch LoadBlueprint in 665ms
INFO - Loading BuildingTiles.txt
INFO - Done Loading BuildingTiles.txt in 7ms
INFO - Loading Conversations.xml
INFO - Done Loading Conversations.xml in 104ms
Starting Game...
Activate Context: {"text":"{{y|{{W|[Esc]}} Back}}","command":"Cancel","hotkey":"Cancel"}
INFO - Subscribed workshop items: 8
MODWARN [2400127933] - Mod using config.json, please convert to manifest.json and check out https://wiki.cavesofqud.com/Modding:Overview for other options to set
INFO - Enabled mods: Clever Girl
INFO - Building script mods
INFO - Done Building script mods in 922ms
INFO - Resetting static caches
INFO - Loading genders
INFO - Done Loading genders in 0ms
INFO - Loading PronounSets.xml
INFO - Done Loading PronounSets.xml in 0ms
Init Console UIs...
Looker INIT!
INFO - Loading Colors.xml
INFO - Done Loading Colors.xml in 1ms
INFO - Done Resetting static caches in 78ms
INFO - Loading genders
INFO - Done Loading genders in 0ms
INFO - Loading PronounSets.xml
INFO - Done Loading PronounSets.xml in 0ms
Init Console UIs...
Looker INIT!
INFO - Loading Colors.xml
INFO - Done Loading Colors.xml in 1ms
INFO - Loading ZoneTemplates.xml
INFO - Done Loading ZoneTemplates.xml in 5ms
INFO - Loading Quests.xml
INFO - Done Loading Quests.xml in 4ms
INFO - Loading ObjectBlueprints.xml
INFO - Loading Mutations.xml
INFO - Done Loading Mutations.xml in 1ms
INFO - Done Loading ObjectBlueprints.xml in 814ms
INFO - Dispatch LoadBlueprint
INFO - Initialize tinkering
INFO - Done Initialize tinkering in 0ms
INFO - Loading Mods.xml
INFO - Done Loading Mods.xml in 0ms
INFO - Initialize liquids
INFO - Done Initialize liquids in 18ms
INFO - Loading PopulationTables.xml
INFO - Done Loading PopulationTables.xml in 18ms
INFO - Done Dispatch LoadBlueprint in 336ms
INFO - Loading BuildingTiles.txt
INFO - Done Loading BuildingTiles.txt in 4ms
INFO - Loading Conversations.xml
INFO - Done Loading Conversations.xml in 33ms
Hot Loading Books...
WARN - Multiple debug commands defined for: hide
WARN - Multiple debug commands defined for: clear
WARN - Multiple debug commands defined for: write
WARN - Multiple debug commands defined for: writeline
WARN - Multiple debug commands defined for: help
WARN - Multiple debug commands defined for: select
WARN - Multiple debug commands defined for: list
WARN - Multiple debug commands defined for: call
WARN - Multiple debug commands defined for: hide
WARN - Multiple debug commands defined for: clear
WARN - Multiple debug commands defined for: write
WARN - Multiple debug commands defined for: writeline
WARN - Multiple debug commands defined for: help
WARN - Multiple debug commands defined for: select
WARN - Multiple debug commands defined for: list
WARN - Multiple debug commands defined for: call
Unknown sprite: 
Activate Context: {"text":"{{y|{{W|[Esc]}} Back}}","command":"Cancel","hotkey":"Cancel"}
INFO - Subscribed workshop items: 8
MODWARN [2400127933] - Mod using config.json, please convert to manifest.json and check out https://wiki.cavesofqud.com/Modding:Overview for other options to set
INFO - Enabled mods: QudUX
INFO - Building script mods
QudUX: Applying pre-game patches...
    QudHistoryFactory...    Patched successfully.
    QudHistoryFactory...    Patched successfully.
    Scores...               Patched successfully.
    XRLCore.NewGame...      Patched successfully.
    AbilityManager...       Patched successfully.
    Look...                 Failed. This patch may not be compatible with the current game version. The option to mark legendary creature locations in journal won't be available from the Look UI.
    GameObject.Move...      Patched successfully.
    Campfire...             Failed. This patch may not be compatible with the current game version. The game's default cooking UI pop-ups will be used instead of QudUX's revamped screens.
    MagneticPulse...        Patched successfully.
    Physics.HandleEvent...  Failed. This patch may not be compatible with the current game version. Some particle text effects may not be shown when movement is prevented.
INFO - Done Building script mods in 602ms
INFO - Resetting static caches
INFO - Loading genders
INFO - Done Loading genders in 0ms
INFO - Loading PronounSets.xml
INFO - Done Loading PronounSets.xml in 0ms
Init Console UIs...
Looker INIT!
INFO - Loading Colors.xml
INFO - Done Loading Colors.xml in 1ms
INFO - Done Resetting static caches in 99ms
INFO - Loading genders
INFO - Done Loading genders in 0ms
INFO - Loading PronounSets.xml
INFO - Done Loading PronounSets.xml in 0ms
Init Console UIs...
Looker INIT!
INFO - Loading Colors.xml
INFO - Done Loading Colors.xml in 1ms
INFO - Loading ZoneTemplates.xml
INFO - Done Loading ZoneTemplates.xml in 4ms
INFO - Loading Quests.xml
INFO - Done Loading Quests.xml in 4ms
INFO - Loading ObjectBlueprints.xml
INFO - Loading Mutations.xml
INFO - Done Loading Mutations.xml in 0ms
INFO - Done Loading ObjectBlueprints.xml in 842ms
INFO - Dispatch LoadBlueprint
INFO - Initialize tinkering
INFO - Done Initialize tinkering in 0ms
INFO - Loading Mods.xml
INFO - Done Loading Mods.xml in 0ms
INFO - Initialize liquids
INFO - Done Initialize liquids in 18ms
INFO - Loading PopulationTables.xml
INFO - Done Loading PopulationTables.xml in 18ms
INFO - Done Dispatch LoadBlueprint in 289ms
INFO - Loading BuildingTiles.txt
INFO - Done Loading BuildingTiles.txt in 4ms
INFO - Loading Conversations.xml
INFO - Done Loading Conversations.xml in 87ms
Hot Loading Books...
WARN - Multiple debug commands defined for: hide
WARN - Multiple debug commands defined for: clear
WARN - Multiple debug commands defined for: write
WARN - Multiple debug commands defined for: writeline
WARN - Multiple debug commands defined for: help
WARN - Multiple debug commands defined for: select
WARN - Multiple debug commands defined for: list
WARN - Multiple debug commands defined for: call
WARN - Multiple debug commands defined for: hide
WARN - Multiple debug commands defined for: clear
WARN - Multiple debug commands defined for: write
WARN - Multiple debug commands defined for: writeline
WARN - Multiple debug commands defined for: help
WARN - Multiple debug commands defined for: select
WARN - Multiple debug commands defined for: list
WARN - Multiple debug commands defined for: call
exTextureInfo must be instantiated using the ScriptableObject.CreateInstance method instead of new exTextureInfo.
System.AggregateException: One or more errors occurred. ---> System.NullReferenceException: Object reference not set to an instance of an object
  at QudUX.Harmony_Patches.Patch_Kobold_SpriteManager.Postfix (System.String Path, exTextureInfo& __result) [0x00000] in C:\Program Files (x86)\Steam\steamapps\workshop\content\333640\1804499742\Harmony Patches\Patch_Kobold_SpriteManager.cs:16 
  at (wrapper dynamic-method) Kobold.SpriteManager.Kobold.SpriteManager.GetTextureInfo_Patch1(string)
  at Kobold.SpriteManager.GetUnitySprite (System.String path) [0x00000] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at SaveManagementRow.setData (XRL.UI.Framework.FrameworkDataElement data) [0x0005d] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at XRL.UI.Framework.FrameworkScroller.SetupPrefab (XRL.UI.Framework.FrameworkUnityScrollChild newChild, XRL.UI.Framework.ScrollChildContext context, XRL.UI.Framework.FrameworkDataElement data, System.Int32 index) [0x0002c] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at XRL.UI.Framework.FrameworkScroller.BeforeShow (XRL.CharacterBuilds.EmbarkBuilderModuleWindowDescriptor descriptor, System.Collections.Generic.IEnumerable`1[T] selections) [0x0044f] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at Qud.UI.SaveManagement.Show () [0x000d0] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at Qud.UI.SaveManagement+<ContinueMenu>d__15.MoveNext () [0x000a7] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
   --- End of inner exception stack trace ---
  at System.Threading.Tasks.Task.ThrowIfExceptional (System.Boolean includeTaskCanceledExceptions) [0x00011] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at System.Threading.Tasks.Task`1[TResult].GetResultCore (System.Boolean waitCompletionNotification) [0x0002b] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at System.Threading.Tasks.Task`1[TResult].get_Result () [0x0000f] in <695d1cc93cca45069c528c15c9fdd749>:0 
  at XRL.Core.XRLCore._Start () [0x00835] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at XRL.Core.XRLCore._ThreadStart () [0x0000a] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
---> (Inner Exception #0) System.NullReferenceException: Object reference not set to an instance of an object
  at QudUX.Harmony_Patches.Patch_Kobold_SpriteManager.Postfix (System.String Path, exTextureInfo& __result) [0x00000] in C:\Program Files (x86)\Steam\steamapps\workshop\content\333640\1804499742\Harmony Patches\Patch_Kobold_SpriteManager.cs:16 
  at (wrapper dynamic-method) Kobold.SpriteManager.Kobold.SpriteManager.GetTextureInfo_Patch1(string)
  at Kobold.SpriteManager.GetUnitySprite (System.String path) [0x00000] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at SaveManagementRow.setData (XRL.UI.Framework.FrameworkDataElement data) [0x0005d] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at XRL.UI.Framework.FrameworkScroller.SetupPrefab (XRL.UI.Framework.FrameworkUnityScrollChild newChild, XRL.UI.Framework.ScrollChildContext context, XRL.UI.Framework.FrameworkDataElement data, System.Int32 index) [0x0002c] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at XRL.UI.Framework.FrameworkScroller.BeforeShow (XRL.CharacterBuilds.EmbarkBuilderModuleWindowDescriptor descriptor, System.Collections.Generic.IEnumerable`1[T] selections) [0x0044f] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at Qud.UI.SaveManagement.Show () [0x000d0] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at Qud.UI.SaveManagement+<ContinueMenu>d__15.MoveNext () [0x000a7] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 <---
Exiting main thread due to stopped game thread.Stopped
Stopping...
Shutting down Steam...
InvalidOperationException: Callback dispatcher is not initialized.
  at Steamworks.CallbackDispatcher.RunFrame (System.Boolean isGameServer) [0x00007] in <ace98fcd411f4c0f8def1928a3818910>:0 
  at Steamworks.SteamAPI.RunCallbacks () [0x00000] in <ace98fcd411f4c0f8def1928a3818910>:0 
  at SteamManager.Update () [0x0000c] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at GameManager.OnUpdate () [0x00f51] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 
  at QupKit.BaseGameController`1[T].Update () [0x00000] in <72242bdb066e4a2e8f0e7f9015dfc07d>:0 

Stopping...
Shutting down Steam...
Setting up 3 worker threads for Enlighten.
  Thread -> id: 16ec -> priority: 1 
  Thread -> id: c90 -> priority: 1 
  Thread -> id: 2f94 -> priority: 1 
Stopping...
Shutting down Steam...

```

**Comment by communist-funny-man** [comment]
Here I leave my player log

**Comment by egocarib** [comment]
Unsure if this was fixed by latest updates (#24) or if it is still a potential issue. Should probably investigate to improve/strengthen the code that caused this exception.

**Comment by egocarib** [comment]
Similar report from Steam recently:
![image](https://github.com/egocarib/CavesOfQud-QudUX-v2/assets/3429497/7b85b0de-e896-42e4-bcd3-cc2a1e5b95a0)
